tips/025_tip_create_new_authority.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: How to create a new Authority
+sitemap:
+priority: 0.1
+lastmod: 2018-10-05T18:20:00-00:00
+---
+# How to create a new Authority
+
+Tip submitted by @Tonterias
```

Comment:
Can you put a link to your Twitter (or GitHub?) handle? It's probably better for you

Reply:
Hi Julien,


Don´t worry, it is ok like that. I'm not a programmer and I just want to learn. 

If I could ask you for something I would like to promote www.JHipsterpress.com as a LIVE learning project, for other people who are starting using Jhipster, like me. Books and examples are great, but not enough for rookies like me. We need to see the whole thing working locally in order to learn.

Have you seen it http://www.jhipsterpress.com/#/ (admin/**admin2**)? (it takes 30+ secs to wake up at Heroku). Please, first check the ReadMe file - Starting Point solutions (https://github.com/Tonterias/JhipsterPress ) with all the tips I’ve made. Those are the stones that I’ve found, while learning, that are not explained in the books and which other newbies might also suffer with. Thanks to Jon Rudell I did not throw the towel.

If I could get some help from other newbies, I would like to move the proyect forward. I think something like this (in the New Online Trainings MOOCs) and a couple of tweets would help in getting one or two learning collaborators:

**Open source collaborative proyect: learn by doing with www.jhipsterpresss.com a WordPress made with Jhipster by the community._**

Anyway, I always wanted to learn how to program a site and Jhipster let me do that. 

Thanks for Jhipster, Julien.

PD: Besides, this is my second PR and I really suffer each time that I try to make one :-D

Reply:
Hi, no problem, I can tweet about it or have links on the main website, etc, exactly like we did for Dan Cancro's big sample project or for all the Moocs about JHipster.
But if you want to write tutorials or examples, we can also give you some space on the main website.

Reply:
Hi Julien,

If I can be of any help to Jhipster, let me know. I'll be glad to help. 

I learned a lot thanks to JHipster. I did JHipsterPress because:

1) I thought it would be an interesting exercise to anyone who was startig (and because I could not follow the 21-points example :-). 
2) I wanted to make something useful as a website (that could be used by people afterwars, even by the Jhipster community) so the WordPress make sense and 
3) because if it can be extended, the learning can reach new highs (with microservices, chat, etc...)

I though I was going to find more people interested, but I did not, but I do not think people know about JHipsterPress yet.

If you think that I'm sharing your goals, I will be glad to get some help for the proyect.

Thanks for your time.

Miguel